app.js

```diff
@@ -1,19 +1,14 @@
 const express = require("express");
 const app = express();
-const {
-  getTopics,
-  
-} = require("./controllers/index.controllers");
-const {
-  
-  handleServerErrors,
-  handleNonExistentPath,
-} = require("./errors/index");
+app.use(express.json())
```

Comment:
Are we using this middleware anywhere yet? If not, it can be removed until we need it 🙂